### PR TITLE
removing roles from personas

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/715)

### CODE REVIEW

File: seeds/data/employees_roles.json [hunk 1]
```diff
@@ -32,9 +32,6 @@
     "code": "fsofficer"
   },
   {
-    "perdet_seq_num": 8,
```
Comment (elizabeth-jimenez) [inline seeds/data/employees_roles.json:35]: Wendy 🦋

File: seeds/data/employees_roles.json [hunk 2]
```diff
@@ -63,10 +60,6 @@
     "perdet_seq_num": 13,
     "code": "AO"
   },
-  {
-    "perdet_seq_num": 13,
```
Comment (elizabeth-jimenez) [inline seeds/data/employees_roles.json:67]: Abby 🦄

**Review by scott062 — COMMENTED** [review]
LIne 55 - We can remove the BMTSOD from AO

File: seeds/data/employees_roles.json [hunk 3]
```diff
@@ -55,18 +51,10 @@
     "perdet_seq_num": 12,
     "code": "PT"
```
Comment (scott062) [inline seeds/data/employees_roles.json:52]: This needs to be changed to `FSBid_Org_Bidders`

**Review by scott062 — COMMENTED** [review]
Admin needs added
`FSBid_Org_Bidders` 
`fsofficer` 
`FSBid_Org_Capsule`

**Review by scott062 — APPROVED** [review]
👩‍🍳 👩‍🎨 🤌

**Pull request merged**